web/crux/src/domain/utils.ts

```diff
@@ -61,6 +61,17 @@ export const toTimestamp = (date: Date): Timestamp => {
   return { seconds, nanos }
 }
 
+export const fromTimestamp = (timestamp: Timestamp): Date => {
```

Comment:
We don't have this mapper already somewhere else?

Reply:
didn't found it. we may have it in the ui code.